dd-sdk-android-core/src/main/kotlin/com/datadog/android/api/feature/FeatureContextUpdateListener.kt

```diff
@@ -0,0 +1,22 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.api.feature
+
+import androidx.annotation.AnyThread
+
+/**
+ * Listener for feature context updates.
+ */
+fun interface FeatureContextUpdateListener {
+
+    /**
+     * Called when the context for a feature is updated.
+     * @param event the updated context
+     */
+    @AnyThread
+    fun onContextUpdate(event: Map<String, Any?>)
```

Comment:
In this case, generally speaking, we cannot reuse the same listener object for context updates of the different features.

What if we add a feature name as a first argument? That way we could re-use the same object.

Not an ask or blocker for me, just some brainstorming.